### PR TITLE
Better number normalization fixes #2419

### LINKED ISSUE #2419: Improve casting to number operations in columns with $ or €
Right now, all columns are null after converting a column with null rows and numbers with $ before.

### PR DESCRIPTION
**Pull request by rafatower**
@Kartones please review

### CODE REVIEW

File: spec/models/table/column_typecaster_spec.rb [hunk 1]
```diff
@@ -1,49 +1,50 @@
 # encoding: utf-8
-require 'rspec/core'
```
Comment (rafatower) [inline spec/models/table/column_typecaster_spec.rb]: I basically reused the spec file: the only test was not in the Makefile and was not recoverable so I wrote a useful one and got rid of the rest.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5099/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5101/))

**Comment by Kartones** [comment]
:+1:

**Comment by saleiva** [comment]
bump!

**Comment by Kartones** [comment]
@saleiva Rafa is on vacation until tomorrow, he said in his worklog he'd release this tomorrow :wink:

**Pull request merged**